templates/miq-template-ext-db.yaml

```diff
@@ -192,6 +192,108 @@ objects:
           resources:
             requests:
               storage: "${APPLICATION_VOLUME_CAPACITY}"
+
+
+- apiVersion: apps/v1beta1
+  kind: "StatefulSet"
+  metadata:
+    name: ${NAME}-backend
```

Comment:
This is user-visible. Is "backend" something they would understand? Maybe "engine"? I don't know what would be the best name here.